restli-server/src/main/java/com/linkedin/restli/internal/server/response/RestLiResponseHandler.java

```diff
@@ -63,57 +64,12 @@
   private final MethodAdapterRegistry _methodAdapterRegistry;
   private final ErrorResponseBuilder _errorResponseBuilder;
 
-  public RestLiResponseHandler()
-  {
-    this(new ErrorResponseBuilder());
-  }
-
-  public RestLiResponseHandler(ErrorResponseBuilder errorResponseBuilder)
-  {
-    this(new MethodAdapterRegistry(errorResponseBuilder), errorResponseBuilder);
-  }
-
   public RestLiResponseHandler(MethodAdapterRegistry methodAdapterRegistry, ErrorResponseBuilder errorResponseBuilder)
   {
     _methodAdapterRegistry = methodAdapterRegistry;
     _errorResponseBuilder = errorResponseBuilder;
   }
 
-  /**
-   * @deprecated Use appropriate constructors.
```

Comment:
Don't remove this just yet. Leave it deprecated. We can remove all of the deprecated stuff together in a follow-up PR.